crates/nu-cli/tests/commands/cd.rs

```diff
@@ -85,9 +85,26 @@ fn filesystem_change_current_directory_to_parent_directory() {
     })
 }
 
+#[test]
+fn filesystem_change_current_directory_to_two_parents_up_using_multiple_dots() {
+    Playground::setup("cd_test_6", |dirs, sandbox| {
+        sandbox.within("foo").mkdir("bar");
+
+        let actual = nu!(
+            cwd: dirs.test().join("foo/bar"),
+            r#"
+                cd ...
+                pwd | echo $it
+            "#
+        );
+
+        assert_eq!(PathBuf::from(actual), *dirs.test());
+    })
+}
+
 #[test]
 fn filesystem_change_current_directory_to_parent_directory_after_delete_cwd() {
-    Playground::setup("cd_test_5_1", |dirs, sandbox| {
+    Playground::setup("cd_test_7", |dirs, sandbox| {
```

Comment:
I should look into the purpose of this name, and see if we can eliminate the need for it 😅

Reply:
Looks like it's the isolated folder for that test, I guess it may be removed if it generated a random name, but it would break some tests that depend on that specific folder, it will need a design change that would inhibit the use of getting out the isolated folder forcing to use only sandbox and files from `tests` folder.